packages/win-eventlog/win-eventlog.0.4/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "Log via the Windows event log from OCaml programs"
+description: """
+A low-level example:
+
+```ocaml
+let log = Eventlog.register "Mirage.exe" in
+let category = 0 and event = 1 in
+Eventlog.report log `Success category event [|
+  "insertion string 1";
+  "insertion string 2";
+|]
+```
+"""
+maintainer: ["dave@recoil.org"]
+authors: ["David Scott"]
+license: "ISC"
+homepage: "https://github.com/mirage/ocaml-win-eventlog"
+doc: "https://mirage.github.io/ocaml-win-eventlog/"
+bug-reports: "https://github.com/mirage/ocaml-win-eventlog/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.08"}
+  "result"
```

Comment:
For next release, this package can be omited (and its corresponding `(libraries result)`) given the `Result` module is already in OCaml >= 4.08

Reply:
Good tip, thanks!